lib/models/asyncapi.js

```diff
@@ -234,6 +234,121 @@ class AsyncAPIDocument extends Base {
   traverseSchemas(callback, schemaTypesToIterate) {
     traverseAsyncApiDocument(this, callback, schemaTypesToIterate);
   }
+
+  /**
+   * Converts a valid AsyncAPI document to a JavaScript Object Notation (JSON) string.
+   * A stringified AsyncAPI document using this function should be parsed via the AsyncAPIDocument.parse() function - the JSON.parse() function is not compatible.
```

Comment:
Do you think we maybe should provide a more in-depth explanation in terms of why it is needed? 🤔

Reply:
Ok, I will describe that functionality in Readme.md :)